src/common/net.cc

```diff
@@ -337,6 +370,41 @@ namespace Pistache
 
     void Address::init(const std::string& addr)
     {
+        // Handle unix domain addresses separately.
+        if (isUnixDomain(addr))
+        {
+            struct sockaddr_un unAddr;
```

Comment:
Please initialize any `struct sockaddr_un` used. Citing the [POSIX standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/sys_un.h.html):

> Although applications are required to initialize all members (including any non-standard ones) of a sockaddr_in6 structure (see [<netinet/in.h>](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/netinet_in.h.html)), the same is not required for the sockaddr_un structure, since historically many applications only initialized the standard members. Despite this, applications are encouraged to initialize sockaddr_un structures in a manner similar to the required initialization of sockaddr_in6 structures.

You should use `struct sockaddr unAddr = {}`, as per POSIX:

> Although it is common practice to initialize a sockaddr_in6 structure using:
> 
>     struct sockaddr_in6 sa;
>     memset(&sa, 0, sizeof sa);
> 
> this method is not portable according to this standard, because the structure can contain pointer or floating-point members that are not required to have an all-bits-zero representation after default initialization. Portable methods make use of default initialization; for example:
> 
>     struct sockaddr_in6 sa = { 0 };

`= {}` is recommended as opposed to `= { 0 }` since its a bit safer to use than the latter. For details, see C23's [N2900](https://www.open-std.org/jtc1/sc22/wg14/www/docs/n2900.htm) accepted proposal.

Reply:
Done.